tests/Livewire/Order/OrderPositionsTest.php

```diff
@@ -146,4 +140,47 @@ public function test_can_show_related_columns()
         $this->assertArrayHasKey('order.uuid', $component->get('data.data.0'));
         $this->assertEquals($this->order->uuid, $component->get('data.data.0')['order.uuid']);
     }
+
+    public function test_quick_add_order_position()
+    {
+        PriceList::query()->update(['is_default' => false]);
```

Comment:
add where constraint to only update price lists which are actually is_default = true.